apps/antalmanac/src/stores/calendarizeHelpers.ts

```diff
@@ -49,7 +49,10 @@ export function calendarizeCourseEvents(currentCourses: ScheduleCourse[] = []):
                         title: `${course.deptCode} ${course.courseNumber}`,
                         courseTitle: course.courseTitle,
                         locations: meeting.bldg.map(getLocation).map((location: Location) => {
-                            return { ...location, days: meeting.days === null ? undefined : meeting.days };
+                            return {
+                                ...location,
+                                days: meeting.days === null ? undefined : COURSE_WEEK_DAYS[dayIndex],
```

Comment:
This seems a bit misleading, because `meeting.days` shouldn't be set to null manually? You didn't change this code, but it would be more appropriately captured as 
```suggestion
                                ...(meeting.days && { days: COURSE_WEEK_DAYS[dayIndex] })
```

Logic: 
The statement will either return something falsy (null, undefined, false, etc.) or an object with the `days` property.
When you destructure a falsy object, it doesn't return anything. When you destructure an actual object, its properties get inherited by the parent object

```ts
const myObject = { hello: 'world', ...(false) }
// myObject = { hello: 'world' }

const myOtherObject = { hello: 'world', ...({ goodbye: 'earth' }) }
// myOtherObject = { hello: 'world', goodbye: 'earth' }
```

Reply:
Makes sense. Glad to know this!